### ISSUE TITLE
gnome-secrets: Fail to start, missing module pwquality

### ISSUE THREAD

**Issue by onny** [NixOS/nixpkgs#170307]
### Describe the bug
When trying to start gnome-secrets it fails with the message:
``` 
ModuleNotFoundError: No module named 'pwquality'
```

### Steps To Reproduce
1. nix-shell -p gnome-secrets --run "secrets"

### Additional context
``` 
Traceback (most recent call last):
  File "/nix/store/hlksjck2iv25ss95mmciqfgr0rzkm08w-gnome-secrets-6.1/bin/.secrets-wrapped", line 67, in <module>
    main()
  File "/nix/store/hlksjck2iv25ss95mmciqfgr0rzkm08w-gnome-secrets-6.1/bin/.secrets-wrapped", line 58, in main
    from gsecrets.application import Application
  File "/nix/store/hlksjck2iv25ss95mmciqfgr0rzkm08w-gnome-secrets-6.1/lib/python3.9/site-packages/gsecrets/application.py", line 10, in <module>
    from gsecrets.widgets.mod import load_widgets
  File "/nix/store/hlksjck2iv25ss95mmciqfgr0rzkm08w-gnome-secrets-6.1/lib/python3.9/site-packages/gsecrets/widgets/mod.py", line 6, in <module>
    from gsecrets.password_generator_popover import PasswordGeneratorPopover
  File "/nix/store/hlksjck2iv25ss95mmciqfgr0rzkm08w-gnome-secrets-6.1/lib/python3.9/site-packages/gsecrets/password_generator_popover.py", line 8, in <module>
    from gsecrets.password_generator import generate as generate_pwd
  File "/nix/store/hlksjck2iv25ss95mmciqfgr0rzkm08w-gnome-secrets-6.1/lib/python3.9/site-packages/gsecrets/password_generator.py", line 7, in <module>
    import pwquality
ModuleNotFoundError: No module named 'pwquality'
```

### Notify maintainers
@mvnetbiz

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.17.3-gnu, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0`
 - channels(root): `"nixos-21.11.337184.a318a09a96a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by onny** [comment]
Also gnome-secrets 6.4 is available. Updating it to the latest version doesn't solve the issue :(

**Comment by mvnetbiz** [comment]
Fixed by #170479

**Comment by onny** [comment]
Thank you @mvnetbiz for your work!
